packages/synapse-react-client/src/synapse-queries/dataaccess/useAccessRequirements.ts

```diff
@@ -163,24 +166,65 @@ export function useCreateLockAccessRequirement(
   })
 }
 
+function getAccessRequirementStatusQueryOptions<
```

Comment:
Pulled this out to reuse in both `useGetAccessRequirementStatus` (one) and `useGetAccessRequirementStatuses` (many)